### PR TITLE
fix: include self in member list & count in Rooms

### PR DESCRIPTION
**Pull request by suprko**
#### Relevant info and issue/PR links 
https://github.com/PeerioTechnologies/peerio-icebear/pull/100
https://app.clubhouse.io/peerio/story/1055/sdk-add-function-to-get-all-chat-participants-including-self
https://app.clubhouse.io/peerio/story/915/desktop-rooms-screen-avatar

#### Testing instructions  
This affects any Room view:
1. Member count at the top includes "self" user
2. Avatars at the beginning of Room history includes self
3. Member list at the right includes self (admin menu hidden on self)

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: src/ui/chat/ChatView.jsx [hunk 1]
```diff
@@ -150,7 +150,7 @@ class ChatView extends React.Component {
                                     tooltip={t('title_Members')}
                                     tooltipPosition="bottom"
                                     onClick={this.toggleSidebar} />
-                                {chat.participants && chat.participants.length ? chat.participants.length : ''}
+                                {chat.allParticipants && chat.allParticipants.length ? chat.allParticipants.length : ''}
```
Comment (anri-asaturov) [inline src/ui/chat/ChatView.jsx]: appParticipants never returns null so first check is not needed

File: src/ui/chat/components/sidebar/MembersSection.jsx [hunk 2]
```diff
@@ -129,8 +132,8 @@ class MembersSection extends React.Component {
                         : null
                     }
                     <List>
-                        {chat.joinedParticipants
-                            ? chat.joinedParticipants.map(
+                        {chat.allJoinedParticipants
```
Comment (anri-asaturov) [inline src/ui/chat/components/sidebar/MembersSection.jsx]: Same here, renove existence check - it always returns an array.

**Comment by anri-asaturov** [comment]
@ebilg ready for testing

**Comment by ebilg** [comment]
self has been added to dms, should be just rooms

**Comment by suprko** [comment]
@ebilg fixed

**Comment by ebilg** [comment]
looks good

**Pull request merged**